server/graphql/v1/mutations/collectives.js

```diff
@@ -513,10 +514,26 @@ export async function archiveCollective(_, args, req) {
     return updatedCollective;
   }
 
-  // TODO: cascade deactivation to EVENTs and PROJECTs?
+  // `changeHost` will recursively check children and unhost them
```

Comment:
So this is gonna fail if a children has a balance. Today the message is not clear and are confused why they still have a balance. https://opencollective.slack.com/archives/CJ2ERRZQ8/p1664980298922799

Reply:
@znarg I introduced a commit to address that: https://github.com/opencollective/opencollective-api/pull/8027/commits/48661938924261e6420fc73d4109140a1f980040